cat-boot-report-pdf/src/test/java/cc/catalysts/boot/report/pdf/impl/DemoTest.java

```diff
@@ -37,8 +50,9 @@ public void demo() throws Exception {
                 .addPadding(10)
                 .startTable()
                 .addColumn("COL1", 2).addColumn("COL2", 2).addColumn("COL3", 4)
-                .createRow().withValues("x1", "x2", "x3")
-                .createRow().withValues("y1", "y2", "y3")
+                .createRow().withValues(new ReportTextBox(styleSheet.getBodyText(), styleSheet.getLineDistance(), "x1"), new ReportImage(img, img.getWidth(), img.getHeight()), new ReportTextBox(styleSheet.getBodyText(), styleSheet.getLineDistance(), "x3"))
+                .createRow().withValues(new ReportTextBox(styleSheet.getBodyText(), styleSheet.getLineDistance(), "y1"), new ReportTextBox(styleSheet.getBodyText(), styleSheet.getLineDistance(), "y3"), new ReportTableBuilderImpl(styleSheet, null)
+                        .addColumn("1", 1).addColumn("2", 3).createRow().withValues("z1", "z2").build())
```

Comment:
please revert back to the old string based test and write a new one for the new api